source/agora/common/Amount.d

```diff
@@ -260,11 +260,27 @@ public struct Amount
     ***************************************************************************/
 
     pragma(inline, true)
-    public void percentage (ubyte percentage)
+    public bool percentage (ushort pct)
```

Comment:
The return value is only used in `mustPercentage`. Why don't you call the `isValid` with `assert`?

Reply:
percentage and mustPercantage methods can be called by anyone and they behave differently:
percentage - returns false if invalid
mustPercantage - asserts if invalid

hence the `must` prefix

this is the same behaviour as with mustSub/sub and mustAdd/add